mobile/src/main/res/values/strings.xml

```diff
@@ -68,6 +68,8 @@
     <!-- Error messages -->
     <string name="error_no_url">Please check that openHAB is running or configure openHAB URL or remote URL</string>
     <string name="error_empty_sitemap_list">openHAB returned empty sitemap list</string>
+    <string name="error_sitemap_generic_load_error">The sitemap could not be loaded. The following error occurred: %1$s</string>
+    <string name="error_sitemap_not_found">The sitemap could not be found on the server. Please select another one from the left navigation menu.</string>
```

Comment:
`from the navigation menu on the left`?

(There is only one drawer, so better not make it sound like there's more than one)

Reply:
ok

Reply:
In my case the error occured, because I edited the sitemap and made a syntax error while it was open on the client.

Are other http errors even possible at this stage? Maybe change the 404 error string to "Error while reloading sitemap. See server log for more info"?

Also: When using "left" you should add a comment so it will be translated to "right" in rtl languages.

Reply:
> Maybe change the 404 error string to "Error while reloading sitemap. See server log for more info"?

IMHO the absence of the expected sitemap will be sufficient to make people look into the server log ;-) A sitemap might be missing e.g. also due to it being (accidentally) deleted, not necessarily due to a syntax error.